Assets/Scripts/LevelGeneration/MapGenerator.cs

```diff
@@ -185,6 +190,60 @@ int NumberOfNeighbors(Vector2 pos)
         return ret;
     }
 
+    public void GenerateWalls(float radius, Vector3 centerPoint, Quaternion instantiatedRotation)
+    {
+        // convert the rotation degrees to radians.
```

Comment:
I think it might be nice to have a quick paragraph explaining this function bc I feel like we'll forget how the math works